resizeobserver/index.html

```diff
@@ -0,0 +1,62 @@
+---
+feature_name: ResizeObserver
+chrome_version: 54
+feature_id: 5705346022637568
+---
+
+<h3>Background</h3>
+<p>
+  The <a href="https://wicg.github.io/ResizeObserver/">ResizeObserver API</a>
+  provides an API to get notified whenever an element changes its size.
+</p>
+<p>
+  This sample illustrates how to use <code>ResizeObserver</code> to implement
+  [media query](https://developer.mozilla.org/en-US/docs/Web/CSS/Media_Queries/Using_media_queries)
+  -like behavior on a per-element basis.
+<p>
+  The <a href="https://developers.google.com/web/updates/2016/10/resizeobserver"><em>Google Developers Web Updates</em></a>
+  article provides more details on <code>ResizeObserver</code>.
+</p>
+
+{% capture initial_output_content %}
+<p>
+  On browsers that support <code>ResizeObserver</code>, you should see the border
+  radius of the second box change when resizing the window.
+</p>
+<style>
+.boxes {
```

Comment:
You've still got the aforementioned problem where the maximum size imposed on the `<body>` element by the default template's styles leads to a situation in which this container won't actually change size until the viewport gets below 800px or so. I.e. if your initial viewport is 1000px, and you resize the viewport to 900px, nothing happens, defeating the point of this sample.

Here's a suggestion for changing the styles so that the size of this container always takes the viewport width into account, so that any resizing will lead to the `ResizeObserver` triggering:

```css
.boxes {
  display: flex;
  flex-wrap: wrap;
  width: 40vw;
}
.box {
  height: 200px;
  flex-shrink: 1;
  flex-grow: 1;
  margin: 10px;
  box-sizing: border-box;
  border: 3px solid black;
}
```

(It adds a `width` to `.boxes`, and removes the `flex-basis` from `.box`.)